core/new-gui/src/app/workspace/service/drag-drop/drag-drop.service.ts

```diff
@@ -179,6 +209,7 @@ export class DragDropService {
 
     // add the operator JointJS element to the paper
     tempGhostModel.addCell(operatorUIElement);
+    this.handleMouseMove();
```

Comment:
Move this to `handleOperatorStartDrag` and rename this method to something like `handleOperatorRecommendationOnDrag`

Reply:
Fixed it.